src/app/shared/entry-actions/workflow-actions.component.html

```diff
@@ -32,4 +32,15 @@
   >
     {{ 'Refresh' }}
   </button>
+  <button
+    *ngIf="workflow?.deletable"
```

Comment:
Similar to Denis' comment:
> Wonder if it would make sense to show it greyed out for non-deletable entries but with explanation. Or maybe just a "i" info type building

I personally prefer showing the greyed out button and providing information to the user about why they can't delete. This way, they know that deleting is an option under the correct circumstances

Reply:
> Similar to Denis' comment:
> 
> > Wonder if it would make sense to show it greyed out for non-deletable entries but with explanation. Or maybe just a "i" info type building
> 
> I personally prefer showing the greyed out button and providing information to the user about why they can't delete. This way, they know that deleting is an option under the correct circumstances

Understood, but here's an argument for the button just disappearing:
An entry can only be deleted if it's never been published.  In other words, once an entry is published, it will never again be deletable, there's nothing the user can do to make it so.  So, the corresponding greyed-out button will sit in the UI next to the never-again-deletable entry, consuming space and attention, taunting the user about what might have been, for eternity.

Reply:
> taunting the user about what might have been, for eternity

:laughing: 
This was kinda my thoughts behind the "info" 
![Screenshot from 2023-07-26 15-04-37](https://github.com/dockstore/dockstore-ui2/assets/1730679/6a8c82ec-6549-403f-a356-7c69db787130)
Because it's less of an actionable thing but more of a BTW

Reply:
> never-again-deletable entry, consuming space and attention, taunting the user about what might have been, for eternity.

Good point :grin: I like Denis' "info" idea too. 

Even if we don't keep the button/info, it would be worth documenting this new functionality since there are conditions that need to be met in order to delete and a screenshot of the delete dialog box might be of interest to a user who can't view it (because they published their workflow, for example)